searchlib/src/vespa/searchlib/common/bitvector.h

```diff
@@ -278,7 +278,9 @@ class BitVector : protected BitWord
      * @param offset            Where bitvector image is located in the file.
      * @param doccount          Number of bits set in bitvector
      */
-    static std::unique_ptr<const BitVector> create(Index numberOfElements, FastOS_FileInterface &file, int64_t offset, Index doccount, ReadStats& read_stats);
+    static std::unique_ptr<const BitVector> create(Index numberOfElements, FastOS_FileInterface &file,
```

Comment:
Consider also adding `entry_size` to the comment parameter list, c.f. `AllocatedBitVector`